src/components/Referrals/referralsHelper.js

```diff
@@ -13,6 +14,12 @@ import { getRootUrl } from "lib/url";
 
 export const REFERRAL_CODE_REGEX = /^\w+$/; // only number, string and underscore is allowed
 export const REGEX_VERIFY_BYTES32 = /^0x[0-9a-f]{64}$/;
+const MAX_DISCOUNT_SHARE = 10000; // 100%
+
+export function removeTrailingZeros(amount) {
+  const regex = /\.?0+$/;
+  return amount.replace(regex, "");
```

Comment:
would `Number(amount)` work in this case? to make it a bit simpler

Reply:
Refactored!